Devscord.DiscordFramework/ReactionsController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Devscord.DiscordFramework.Integration;
+using Discord.WebSocket;
+
+namespace Devscord.DiscordFramework
+{
+    public class ReactionsController
```

Comment:
Nad tą nazwą i namespace bym sie zastanowił
Controller mi tu niezbyt pasuje

Może ReactionsHandler

Reply:
a wy jakieś przestrzeni miałoby to być?

Reply:
@Marcin99b tylko skoro będzie inna nazwa, to nie będzie rejestrowany przez autofaca
musi być końcówka Service lub Controller

Reply:
a może ReactionsHandlerService?

Reply:
albo handler, albo service 
ReactionsService też będzie ok

Reply:
+zawsze do autofaca można dodać żeby rejestrował

ale z tego co pamiętam to już nie ma tego, że musi być Service 
bo już to jest zautomatyzowane od dawna

ale to na 90%, nie jestem pewny

Reply:
Zgadza sie, teraz jest
```csharp
                builder.RegisterAssemblyTypes(asm)
                    .Where(x => x.FullName.StartsWith("Watchman") || x.FullName.StartsWith("Devscord"))
                    .PreserveExistingDefaults()
                    .InstancePerLifetimeScope();
```

Reply:
@gagyn

Reply:
mnie się wydaje że po nazwie rejestruje, czyli końcówka "Service" bo jak inaczej?

Reply:
a to musiałem nie zauważyć tej zmiany

Reply:
przestrzeń Devscord.DiscordFramework.Services będzie dobrą lokalizacją tej klasy?

Reply:
myśle że może być